src/Stratis.Bitcoin/Consensus/ConsensusExceptions.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+
+namespace Stratis.Bitcoin.Consensus
+{
+    public class ConsensusException : Exception
```

Comment:
We already have `ConsensusErrors.cs` with a list of all possible consensus exceptions. Maybe add the exceptions there? 

Otherwise it's not clear what is the difference between the ConsensusError and ConsensusException

Reply:
I'll consolidate later